static/js/settings.js

```diff
@@ -409,7 +409,8 @@ const SettingsScreen = {
         let updateUrl = null, updateVersion = null, updateChecksum = null;
         if (this.availableAddons.has(name)) {
           const available = this.availableAddons.get(name);
-          if (available.version !== addon.version) {
+          const cmp = this.compareSemver(addon.version, available.version);
```

Comment:
The compareSemver function can be replaced with semver.compare

Reply:
Not on the UI side...